README.md

```diff
@@ -1,2 +1,54 @@
 # dataworks-common-logging
 Kotlin utility library to be used in Dataworks applications to ensure common logging format.
+
+## Log formatting
+Dataworks common logging provides opinionated ways to write messages into log files, using [sl4j](http://www.slf4j.org/).
+
+Out of the box, it provides functionality to convert log messages to JSON and appends a number of common fields. These can be found in the `LogField` enum class. Any `LogField` which is not found at runtime gets set to a default value, also defined in that enum. Some example variable values are as follows:
```

Comment:
Not found at runtime from where? ENV_VAR? property.value??? Can we have examples please

Reply:
I was trying to point the users towards the KDoc in that class since it's more specific than the README (and more likely to be updated). I am happy to put a note here about where they are derived from, or specifically refer to the KDoc?

Reply:
Following a chat with Mark, I have added some further details to the README